.travis.yml

```diff
@@ -18,6 +20,12 @@ script:
 # Dump trial log on failure.
 after_failure: "cat _trial_temp/test.log"
 
+# Only run test when committing to master branch.
+branches:
+  only:
+    - master
+    - \d+\.\d+\.\d+
```

Comment:
Could you test it with a prerelease?  (That would also change the regex, I think.)